### DISCUSSION TITLE
Actions tab missing, /actions URL 404, /runs work

### DISCUSSION THREAD

**Discussion by sormuras** [community/community#25821] [Actions]
<p>We’re experiencing an issue with the <code>Actions</code> tab at <a href="https://github.com/junit-team/junit5" rel="nofollow noopener">https://github.com/junit-team/junit5</a></p>
<p>a) the tab is missing (since some days already)</p>
<p>b) the URL <a href="https://github.com/junit-team/junit5/actions" rel="nofollow noopener">https://github.com/junit-team/junit5/actions</a> leads to a <code>404</code> error page (which worked some days ago)</p>
<p>c) individual Actions Runs are browsable, e.g. <a href="https://github.com/junit-team/junit5/runs/277499190" rel="nofollow noopener">https://github.com/junit-team/junit5/runs/277499190</a></p>
<p>d) triggered an issue at our tracker <a href="https://github.com/junit-team/junit5/issues/2089" rel="nofollow noopener">https://github.com/junit-team/junit5/issues/2089</a></p>
<p>Settings of “<a href="https://github.com/junit-team/junit5" rel="nofollow noopener">junit-team/junit5</a>” seem okay.</p>
<p>What can we do?</p>

**Comment by BrightRan** [comment]
<p>Hi <a class="mention" href="/sormuras">@sormuras</a> ,</p>
<p>Please check if Actions has been disabled for your repository. The situation you reported generally happens when Actions disabled for the repository.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/46a5fb3507f63cd606672e8f89d514cf3e310631.png" data-download-href="/uploads/short-url/a4Z2f4yeCVblRvHmI8oQTtc7c1r.png?dl=1" title="settings.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46307996/181093622-98485e82-3f17-435d-bc9d-c3f303271aa4.png" alt="settings.png" data-base62-sha1="a4Z2f4yeCVblRvHmI8oQTtc7c1r" width="690" height="351" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/46a5fb3507f63cd606672e8f89d514cf3e310631_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">settings.png</span><span class="informations">1012×516 23.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by sormuras** [comment]
<p>It’s enabled.</p>
<p>Also we tried turning it off and on again… didn’t help.</p>

**Comment by sormuras** [comment, marked as answer]
<p>Manual fix applied by GitHub team. Thanks!</p>
